holoviews/core/spaces.py

```diff
@@ -788,6 +796,73 @@ def dynamic_redim(obj):
         return Dynamic(redimmed, shared_data=True, operation=dynamic_redim)
 
 
+    def collate(self, streams=[]):
+        """
+        Collation allows collapsing DynamicMaps with invalid nesting
+        hierarchies. This is particularly useful when defining
+        DynamicMaps returning an (Nd)Layout. Collating will split the
+        DynamicMap into an (Nd)Layout of individual DynamicMaps. Note
+        that the Layout should be of consistent length and types for
+        this to work correctly. In order to attach a stream as a source
+        for a particular object in the Layout you may supply either
+        a dictionary or list of lists of streams corresponding to each
+        Element in the Layout.
```

Comment:
I think ``Viewable`` may be right. We rarely use that term though but I don't think I mind it here...